plenum/common/signer_did.py

```diff
@@ -33,11 +47,22 @@ def identifier(self) -> Identifier:
 
     @property
     def verkey(self) -> str:
+        if self._verkey is None:
```

Comment:
Since you always have `abbreviated` set when you follow the comment above, you don't need this and next line